examples/metrics-basic/src/main.rs

```diff
@@ -12,10 +12,11 @@ fn init_meter_provider() -> opentelemetry_sdk::metrics::SdkMeterProvider {
     let reader = PeriodicReader::builder(exporter).build();
     let provider = SdkMeterProvider::builder()
         .with_reader(reader)
-        .with_resource(Resource::new([KeyValue::new(
-            "service.name",
-            "metrics-basic-example",
-        )]))
+        .with_resource(
+            Resource::builder_empty()
```

Comment:
I think we can consistently use Resource::builder in all examples, as that would be normal path most users need. builder_empty() is only for the advanced users who want to avoid the default resources.

Reply:
okay will update, I was keeping consistent to how it was before